server/web/models.py

```diff
@@ -0,0 +1,30 @@
+from . import db
+from werkzeug.security import generate_password_hash, check_password_hash
+
+
+class Account(db.Model):
+    __tablename__ = 'account'
+    id = db.Column(db.Integer, primary_key=True)
+    companyName = db.Column(db.String(64), unique=True)
+    createDate = db.Column(db.DateTime)
+    active = db.Column(db.Boolean)
+
+    def __repr__(self):
+        return '<Account %r>' % self.companyName
+
+class User(db.Model):
+    __tablename__ = 'users'
+    id = db.Column(db.Integer, primary_key=True)
+    acct_id = db.Column(db.String(64)) #still need set this to be foreign key
```

Comment:
Per [documentation](http://flask-sqlalchemy.pocoo.org/2.3/models/#one-to-many-relationships)
```python
db.Column(db.Integer, db.ForeignKey('account.id'), nullable=False) # table_name.column_name as a string basically
```

Data type must be consistent. `id` is an integer on the `Account` model. Pass into foreign key the table name & column name as a string